src/integration/pages/idxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,134 @@
+import { runCommonCrossPlatformTests, runMainHeadingTests } from '../../common';
+
+export default () => {
+  runCommonCrossPlatformTests();
+  runMainHeadingTests();
+
+  describe('Sections', () => {
+    const sections = document.querySelectorAll('section');
+
+    if (sections) {
+      sections.forEach(section => {
+        const h2El = section.querySelector('h2');
+        it('should be in the document', () => {
+          expect(section).toBeInTheDocument();
+        });
+
+        it('should have a headline', () => {
+          expect(h2El).toBeInTheDocument();
+        });
+
+        it('should have a headline with text', () => {
+          expect(h2El.textContent).toBeTruthy();
+        });
+
+        it('should match headline text', () => {
+          expect(h2El.textContent).toMatchSnapshot();
+        });
+      });
+    }
+  });
+
+  describe('Index Alsos', () => {
+    const topStories = document.querySelector(
+      '[aria-labelledby="Top-stories"]',
+    );
+
+    if (topStories) {
```

Comment:
I think we can remove this check as @karinathomasbbc suggested https://github.com/bbc/simorgh/pull/6802#discussion_r438924523